lib/tasks/penn_lib.rake

```diff
@@ -120,6 +120,46 @@ namespace :pennlib do
 
   end
 
+  namespace :subject_specialists do
+    desc 'pulls the current list of subject specialists and their photos from Drupal'
+    task :update => :environment do |t, args|
```

Comment:
Is this task used in development to populate the JSON for the librarian data and the corresponding images? we also include those files in the repo. it could be nice to just run this task in production to update the info without having to redeploy everything. that might mean we shouldnt include (instead ignore) those files in the repo?

Reply:
after talking with @tampakis we'll keep this this way for now but look at how the rake task performs in a production-like environment before making any additional changes